### PR TITLE
Add com.github.gyunaev.spivak

### PR DESCRIPTION
**Pull request by cosimoc**
A QT-based karaoke app.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.github.gyunaev.spivak

File: com.github.gyunaev.spivak.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
+    "app-id": "com.github.gyunaev.spivak",
+    "command": "spivak",
+    "rename-desktop-file": "spivak.desktop",
+    "rename-icon": "spivak",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "finish-args":[
+	"--socket=x11",
```
Comment (nedrichards) [inline com.github.gyunaev.spivak.json]: if it's socket x11 you should have --share=ipc as well. Also, all Qt apps support wayland these days (although do test with something this 'artisan'

File: com.github.gyunaev.spivak.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	"/include",
+	"/lib/pkgconfig",
+	"/lib64/pkgconfig",
+	"/share/doc",
+	"/share/man",
+	"*.a",
+	"*.la"
+    ],
+    "modules":[
+	{
```
Comment (nedrichards) [inline com.github.gyunaev.spivak.json]: There's a slightly different approach to cld2 over in poedit. Not a requirement but an interesting comparison: https://github.com/flathub/net.poedit.Poedit/blob/master/net.poedit.Poedit.json#L113

File: com.github.gyunaev.spivak.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	    "no-parallel-make": true,
+	    "sources":[
+		{
+		    "type": "git",
+		    "url": "https://github.com/gyunaev/spivak.git",
+		    "tag": "v1.2"
+		},
+		{
+		    "type": "script",
+		    "dest-filename": "autogen.sh",
```
Comment (nedrichards) [inline com.github.gyunaev.spivak.json]: couldn't you just use the qmake buildsystem?

**Comment by nedrichards** [comment]
I see that upstream asks you to turn off breakpad reporting, although I suspect that they could get the symbols from the debug extension if required.

**Comment by nedrichards** [comment]
https://flathub.org/builds/#/builders/1/builds/2396

**Comment by cosimoc** [comment]
@nedrichards Ready for another round.

**Comment by nedrichards** [comment]
bot, build com.github.gyunaev.spivak

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.gyunaev.spivak

**Pull request closed without merging**